t/data/09-schedule_from_file.yaml

```diff
@@ -0,0 +1,51 @@
+products:
+  sle-online-aarch64-*:
+    distri: "opensuse"
+    flavor: "DVD"
+    arch  : "i586"
+    version: '13.1'
+    settings:
+      foo: 1
+  sle-offline-x86_64-*:
+     distri: "sle"
+     flavor: "offline"
+     arch: "x86_64"
+     version: '12-SP5'
+     settings:
+       foo : 2
+machines:
+  64bit:
+    backend: "qemu"
+    priority: 20
+    settings: 
+      ARCH_BASE_MACHINE: "64bit"
+      QEMUCPUS: "2"
+      QEMURAM: "2048"
+      QEMUVGA: "virtio"
+      WORKER_CLASS: "qemu_x86_64"
+  aarch64:
+    backend: "qemu"
+    settings: 
+      QEMU: "aarch64"
+      QEMURAM: 3072
+      UEFI: "1"
+      WORKER_CLASS: "qemu_aarch64"
+job_templates:
+  autoyast_bcache:
+    product: "sle-offline-x86_64-*"
+    machine: "b4bit" 
+    settings:
+      YAML_SCHEDULE: 'schedule/yast/autoyast_bcache.yaml'
```

Comment:
I would suggest not mentioning the `YAML_SCHEDULE` here, as it is a os-autoinst-distri-opensuse specific feature and not one in openQA in general.